### PR TITLE
Make sure jdk exists before running tests that use it.

### PR DESCRIPTION
**Pull request by smillst**
Also, copy jdk.jar to dist after it is created.

### CODE REVIEW

File: checker/build.xml [hunk 1]
```diff
@@ -1508,6 +1508,8 @@
          <env key="LT_BIN" value="${LT_BIN}"/>
       </exec>
       <move file="jdk/jdk.jar" tofile="jdk/${jdkName}"/>
+        <!-- Copy jars to dist. If dist doesn't exist then the copy task creates it.-->
```
Comment (wmdietl) [inline checker/build.xml]: The indentation seems off.

File: checker/build.xml [hunk 2]
```diff
@@ -1508,6 +1508,8 @@
...
+        <copy file="jdk/${jdkName}" tofile="dist/${jdkName}"/>
```
Comment (wmdietl) [inline checker/build.xml]: This should have `overwrite="true"` and maybe also `failonerror="false"` to account for an already existing file.

Comment (wmdietl) [inline checker/build.xml]: Target `dist-nobuildjdk` at the moment also copies around jdks. Those copies be removed now, right?

Comment (smillst) [reply checker/build.xml]: By default, if the file exists and is older than the copied file, it is overwritten.  Using `overwrite="true"`  overwrites existing files if they are newer than the copy.  I don't think we want that.

Comment (wmdietl) [reply checker/build.xml]: It's what `dist-nobuildjdk` was doing. Fine with me if it works without that option.

**Pull request merged**